libsql-client/src/cell_value.rs

```diff
@@ -0,0 +1,57 @@
+//! `CellValue` represents libSQL values and types.
+//! Each database row consists of one or more cell values.
+
+/// Value of a single database cell
+#[derive(Clone, Debug)]
+pub enum CellValue {
```

Comment:
Maybe it would be better to name this just `Value`? The same structure is also used for parameter values, and there are no other kinds of values that would need to be distinguished from this enum.

Reply:
Yup, makes sense; 0.3.x is already published on crates.io, so let's refactor it as a follow-up and make it part of 0.4.x maybe?